src/views/Collectibles/components/NftGiveawayModal.tsx

```diff
@@ -41,8 +41,10 @@ const getClaimableNft = (profile: Profile): Nft => {
     return null
   }
 
-  const bunnyId = Object.keys(teamNftMap).find((mapBunnyId) => teamNftMap[mapBunnyId] === profile.team.id)
-  return nftList.find((nft) => nft.bunnyId === Number(bunnyId))
+  const identifier = Object.keys(teamNftMap).find(
+    (mapNftIdentifier) => teamNftMap[mapNftIdentifier] === profile.team.id,
+  )
```

Comment:
Wouldn't using `Object.values` make more sense?

```suggestion
  const identifier = Object.values(teamNftMap).find(
    (teamId) => teamId === profile.team.id,
  )
```

Reply:
Not sure this will work. I need to get the nft `identifier`. That will return the team id

Reply:
Ah, yes, you're right, my bad.